src/ui/button/props.js

```diff
@@ -147,37 +152,36 @@ export const DEFAULT_PROPS = {
 // const ALLOWED_COLORS = values(BUTTON_COLOR);
 const ALLOWED_SHAPES = values(BUTTON_SHAPE);
 
-function getDefaultButtonContent() : ButtonContent {
-    // $FlowFixMe
-    return {};
-}
+// function getDefaultButtonContent() : ButtonContent {
```

Comment:
If this needs to be removed, please remove it. Would prefer to not leave unused commented code.

Reply:
removed